extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2048,6 +2048,30 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'scroll': {
```

Comment:
nit: Could you help sort vendors in alphabetical order. And then add a comment that this is only for your amp-access-scroll, no request will be ever sent without that.